x-pack/plugins/apm/public/components/alerting/ui_components/alert_details_app_section/types.ts

```diff
@@ -5,11 +5,11 @@
  * 2.0.
  */
 
-import { Rule } from '@kbn/alerting-plugin/common';
-import { TopAlert } from '@kbn/observability-plugin/public/pages/alerts';
-import { AlertSummaryField } from '@kbn/observability-plugin/public/pages/alert_details/types';
-import { TIME_UNITS } from '@kbn/triggers-actions-ui-plugin/public';
-import { SERVICE_ENVIRONMENT } from '../../../../../common/es_fields/apm';
+import type { Rule } from '@kbn/alerting-plugin/common';
+import type { AlertSummaryField } from '@kbn/observability-plugin/public/pages/alert_details/components/alert_summary';
+import type { TopAlert } from '@kbn/observability-plugin/public/typings/alerts';
```

Comment:
We should export `AlertSummaryField` and `TopAlert` from `@kbn/observability-plugin/public` so if we change these paths in the future, we don't have to get approval from APM (or other solutions).